graylog2-web-interface/src/logic/telemetry/Constants.ts

```diff
@@ -181,6 +181,20 @@ export const TELEMETRY_EVENT_TYPE = {
     STREAM_ITEM_CLONED: 'Stream Item Cloned',
     STREAM_ITEM_RULE_SAVED: 'Stream Item Rule SAVED',
     STREAM_ITEM_DELETED: 'Stream Item Deleted',
+    STREAM_ITEM_DATA_ROUTING_CLICKED: 'Stream Item Data Routing Clicked',
+    STREAM_ITEM_DATA_ROUTING_UPDATE_CLICKED: 'Stream Item Data Routing Update Clicked',
+    STREAM_ITEM_DATA_ROUTING_STREAM_INDEXSET_UPDATE_OPENED: 'Stream Item Data Routing Stream IndexSet Update Opened',
+    STREAM_ITEM_DATA_ROUTING_INDEXER_FILTER_UPDATE_OPENED: 'Stream Item Data Routing Indexer Filter Update Opened',
+    STREAM_ITEM_DATA_ROUTING_INDEXER_FILTER_CREATE_OPENED: 'Stream Item Data Routing Indexer Filter Create Opened',
+    STREAM_ITEM_DATA_ROUTING_DATA_WAREHOUSE_FILTER_CREATE_OPENED: 'Stream Item Data Routing Indexer Filter Create Opened',
+    STREAM_ITEM_DATA_ROUTING_DATA_WAREHOUSE_FILTER_UPDATE_OPENED: 'Stream Item Data Routing Indexer Filter Update Opened',
```

Comment:
Data Warehouse instead of Indexer in the above 2 lines

Reply:
This is meant to be an indexer because you can create filters both for data warehouse and indexer.